wp-content/themes/reconasia/archive.php

```diff
@@ -13,23 +13,34 @@
 <main id="site-content" role="main">
 
 	<?php
-
-		get_template_part( 'template-parts/entry-header' );
+		if (class_exists('ACF') && !is_paged()) {
+			$term = get_queried_object();
+			// vars
+			$featured_post = get_field('featured_post', $term);
+			if ( $featured_post ) {
+
+				echo '<section class="archive__featured">';
+					echo '<h2 class="archive__featured-label">Featured</h2>';
+					foreach( $featured_post as $post ):
+						// Setup this post for WP functions (variable must be named $post).
+						setup_postdata($post);
+						// get_template_part( 'template-parts/block-post-featured' );
+						get_template_part( 'template-parts/block', get_post_type() );
+					endforeach;
+				echo "</section>";
+				// Reset the global post object so that the rest of the page works correctly.
+				wp_reset_postdata();
+			}
+		}
 
 		if ( have_posts() ) {
-
-			reconasia_pagination_number_of_posts();
```

Comment:
Let's keep the stuff for the pagination in here, please!

Reply:
Gotcha! I added this back and for some reason is closing ```</main>``` so is breaking the articles
![is-closing-main](https://user-images.githubusercontent.com/51330062/113630216-81337b80-9635-11eb-9ad9-66f182fd0b96.png)

Reply:
I pushed an adjustment that fixes this. :)